java/kafka/streams/src/main/java/KafkaStreamsExample.java

```diff
@@ -39,8 +53,9 @@ public static void main(String[] args) {
                 .to(config.getTargetTopic(), Produced.with(Serdes.String(), Serdes.String()));
 
         KafkaStreams streams;
+        TracingSystem tracingSystem = KafkaStreamsExample.TracingSystem.forValue(config.getTracingSystem());
 
-        if (System.getenv("JAEGER_SERVICE_NAME") != null)   {
+        if (tracingSystem != null) {
```

Comment:
We need to add the logic we used in the Producer and Consumer examples code for choosing between `jaeger` and `opentracing` here as well